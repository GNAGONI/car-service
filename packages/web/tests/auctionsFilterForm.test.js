import React from 'react'
import renderer from 'react-test-renderer'
import AuctionsFilterForm from 'components/auctionsFilterForm'

it('renders correctly', () => {
  const tree = renderer.create(<AuctionsFilterForm sortByOptions={[]} onSubmit={() => {}} />).toJSON()
  expect(tree).toMatchSnapshot()
})
