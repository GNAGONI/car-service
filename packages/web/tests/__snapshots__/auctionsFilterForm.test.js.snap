// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  action="#"
  className="filter-form"
  onSubmit={[Function]}
>
  <div
    className="input-row"
  >
    <div
      className="form-group"
    >
      <strong
        className="title"
      >
        Filter Results
      </strong>
      <div
        className="form-row"
      >
        <span
          className="label-wrap"
        >
          <label
            htmlFor="search"
          >
            <i
              className="icon-search"
            />
          </label>
        </span>
        <div
          className="input-holder"
        >
          <div
            className="input-wrap"
          >
            <input
              className="form-control"
              id="search"
              onChange={[Function]}
              placeholder="Search by Keywords"
              type="text"
              value=""
            />
          </div>
          <div
            className="btn-holder"
          >
            <button
              className="btn btn-dark"
            >
              Search
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <strong
        className="title"
      >
        Sort by:
      </strong>
      <div
        className="form-row"
      >
        <span
          className="label-wrap"
        >
          <label
            htmlFor="sort"
          >
            <i
              className="icon-filter"
            />
          </label>
        </span>
        <div
          className="input-holder"
        >
          <div
            className="input-wrap"
          >
            <div
              className="css-l7oe9o"
              onKeyDown={[Function]}
            >
              <div
                className="css-1d4rsr1"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <div
                  className="css-hxuxk6"
                >
                  <div
                    className="css-1492t68"
                  >
                    Filter by...
                  </div>
                  <input
                    className="css-14uuagi"
                    disabled={false}
                    id="react-select-2-input"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    readOnly={true}
                    tabIndex="0"
                    value=""
                  />
                </div>
                <div
                  className="css-1wy0on6"
                >
                  <span
                    className="css-rb20kt"
                  />
                  <div
                    aria-hidden="true"
                    className="css-1ep9fjw"
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                  >
                    <svg
                      aria-hidden="true"
                      className="css-19bqh2r"
                      focusable="false"
                      height={20}
                      viewBox="0 0 20 20"
                      width={20}
                    >
                      <path
                        d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="btn-holder"
          >
            <button
              className="btn btn-dark btn-apply"
            >
              Apply
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</form>
`;
